addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -922,6 +922,40 @@ QUnit.test('Attachments that have been unlinked from server should be visually u
     );
 });
 
+QUnit.test('the chatter should be empty when creating a new record after having displayed a chatter for an existing record', async function (assert) {
+    assert.expect(2);
+
+    this.data['res.partner'].records.push({ id: 12 });
+    await this.createView({
+        data: this.data,
+        hasView: true,
+        View: FormView,
+        model: 'res.partner',
+        res_id: 12,
+        arch: `
+            <form>
+                <div class="oe_chatter">
+                    <field name="message_ids"/>
```

Comment:
Don't think we need this since refactoring.

Reply:
It's still our only way to know whether the message list has to be displayed, we didn't change chatter API yet, just removed widget= from it.

Reply:
Ok, ignore comment then.